pkg/eval/testutils.go

```diff
@@ -192,6 +217,15 @@ func (t TestCase) ThrowsMessage(msg string) TestCase {
 	return t.throws(errWithMessage{msg})
 }
 
+// ThrowsCmdExit returns an altered TestCase that requires the source code to
+// throw an ExternalCmdExit error that ignores the pid.
+func (t TestCase) ThrowsCmdExit(err error) TestCase {
```

Comment:
The problem is that the pre-existing `NewExternalCmdExit` function returns an `error`. I'll investigate whether it is appropriate, and makes more sense, for that function to return an `ExternalCmdExit`.

Reply:
I decided to modify the unit test to directly instantiate a `ExternalCmdExit` struct rather than call `NewExternalCmdExit` (which returns an `error` object). Thus allowing me to make this method require a `ExternalCmdExit` rather than a generic `error` object as you requested. Still, it's not clear this is an improvement due to the disconnect with the value returned by `NewExternalCmdExit`.